ykman/driver_ccid.py

```diff
@@ -102,8 +102,9 @@ def __init__(self, connection, name):
         self._pid = _pid_from_name(name)
         try:
             self._read_serial()
-        except APDUError:
-            pass  # Can't read serial or version
+        except APDUError as e:
+            # Can't read serial or version
+            logger.error('Failed to read serial number', exc_info=e)
```

Comment:
Failed to read serial or version? Or is the comment outdated?

Reply:
I wasn't sure, judging by the method name, so I left the comment in place. But it turns out the `_read_serial` method does indeed read both serial number and version, so I'll refactor that.